linux/installer/common/sgx-aesm-service/linksgx.sh

```diff
@@ -35,8 +35,20 @@ if test $(id -u) -ne 0; then
     exit 1
 fi
 
-/usr/bin/getent group sgx_prv &> /dev/null
-if [ $? == "0" ]; then
+if [ -c /dev/sgx_provision -o -c /dev/sgx/provision ]; then
+    /usr/bin/getent group sgx_prv &> /dev/null
+    if [ "$?" != "0" ]; then
+        # Add sgx_prv for dcap driver, which ensures that no matter what 
+        # the order of package installation, aesmd can have access to 
+        # the sgx_provision device file.
+        /usr/sbin/groupadd sgx_prv
+
+        if ! which udevadm &> /dev/null; then
+            return 0
```

Comment:
Better to use `exit 0` instead of `return 0`.

Reply:
Copy that.

Reply:
Fixed.